pkg/addons/addons.go

```diff
@@ -488,3 +500,90 @@ func enableOrDisableAutoPause(cc *config.ClusterConfig, name string, val string)
 
 	return nil
 }
+
+// enableOrDisableGCPAuth enables or disables the gcp-auth addon depending on the val parameter
+func enableOrDisableGCPAuth(cfg *config.ClusterConfig, name string, val string) error {
```

Comment:
this file is starting to get large, how about moving each addons enable code to its own file
addons_gcp_auth.go

(feel free to do same other addons that have special enanble disable)
so this file will be shorter

Reply:
done